ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,212 @@
+                            int ccount = 0;
+                            foreach (Poll poll in pollList)
+                            {
+                                Console.WriteLine($"{ccount}. {poll.Question} {poll.Result}");
+                                ++ccount;
+                            }
+
+                        }
+                        break;
+                    case '3':
+                        int count = 0;
+                        foreach (Poll poll in pollList)
+                        {
+                            Console.WriteLine($"{count}. {poll.Question}");
+                            ++count;
+                        }
+                        Console.WriteLine("Choose question");
+                        int k = Int32.MaxValue;
+                        try
+                        {
+                            k = Convert.ToInt32(Console.ReadLine());
+                        }
+                        catch
+                        {
+
+                        }
+                        if (k != Int32.MaxValue)
+                        {
+                            Console.Clear();
+                            Console.WriteLine(pollList[k].Question);
+                            Console.WriteLine("Are you agree? y/n");
+                            char c = Console.ReadKey().KeyChar;
+                            switch (c)
+                            {
+                                case ('y'):
+                                    pollList[k].Voters.Add(new Voter { User = (pollList[k].Voters.Count + 1), vote = true });
+                                    break;
+                                default:
+                                    pollList[k].Voters.Add(new Voter { User = (pollList[k].Voters.Count + 1), vote = false });
+                                    break;
+                            }
+                            
+                        }
+                        Console.ReadKey();
+                        break;
+                    case '4':
+                        stat = false;
+                        break;
+                }
+
+            }
+        }
+
+        public class Poll
+        {
+            //Lazy implemenation
+            public Poll(string question)
+            {
+                Question = question;
+                Voters = new List<Voter>();
+            }
+           public string Question { get; set; }
+           public List<Voter> Voters { get; set; }
+           private string _result { get; set; }
+           public string Result { 
+                get 
+                {
+                    int positive = 0;
+                    int negative = 0;
+                    foreach(Voter vooter in Voters)
+                    {
+                        if (vooter.vote == true) positive++;else negative++;
+                    }
+                    return $"Voted {Voters.Count} users, where {positive} agree and {negative} disagree";
```

Comment:
because you have 'positive' and 'negative' inside this scope, they always be 1 and 0